app/controllers/pages_controller.rb

```diff
@@ -2,5 +2,6 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [ :home ]
 
   def home
+    @yachts = Yacht.all
```

Comment:
Maybe in home we can show only the last five added?

`Yacht.limit(5).order('id desc')`

Reply:
Yep, the logic I implemented was just to display some items in order to use them in the front-end. I guess we could do that 👌